src/tools/dataOrganizer/drawer.js

```diff
@@ -89,7 +89,8 @@ class Drawer {
   }
   draw(path) {
     try {
-      fs.writeFileSync(path, this.stringImage);
+      const write = fs.existsSync(path) ? fs.writeFileSync : fs.appendFileSync;
```

Comment:
Why not just use append? It creates a new File if it can't find it.

Reply:
Yes, I meant to remove .writeFileSync because it is unnecessary

Reply:
So in this case we need to make file be cleared by either deleting it or writing `''` to it?

Reply:
The idea was to refresh the file when it exists (not just append to it) & create a new file that will refresh on each function call if it doesn't exist.

Reply:
.writeFileSync creates a new file if it does not exist, so you don't need .append

[https://www.geeksforgeeks.org/node-js-fs-writefilesync-method/
](url)

Reply:
Roger that, will approve this.